docs/recipes/github-actions.md

```diff
@@ -0,0 +1,66 @@
+# Using semantic-release with [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+
+For this example, you'll need to setup a [`NPM_TOKEN`](https://docs.npmjs.com/creating-and-viewing-authentication-tokens) to publish your package to NPM registry. GitHub Actions [automatically adds](https://help.github.com/en/articles/virtual-environments-for-github-actions#github_token-secret) [`GITHUB_TOKEN`](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) to be used in workflows.
+
+## Node project configuration
+
+[GitHub Actions](https://github.com/features/actions) supports [Workflows](https://help.github.com/en/articles/configuring-workflows) allowing to test on multiple Node versions and publishing a release only when all test pass.
+
+**Note**: The publish pipeline must run a [Node >= 8 version](../support/FAQ.md#why-does-semantic-release-require-node-version--83).
+
+### `.github/workflows/release.yml` configuration for Node projects
+
+This example is a minimal configuration for [`semantic-release`](https://github.com/semantic-release/semantic-release) with a build running Node 12 when receives a new commit at `master` branch. See [Configuring a workflow](https://help.github.com/en/articles/configuring-a-workflow) for additional configuration options.
+
+```yaml
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  install:
+    name: "Generate release"
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12
+      - name: Install
+        run: npm ci
+      - name: Generate release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
+```
+
+## Pushing `package.json` changes to `master` branch
+
+If you want to keep your `package.json` updated in your code versioning with your released version you could use [`@semantic-release/git`](https://github.com/semantic-release/git) plugin. To use it you'll need to generate a [`GITHUB_TOKEN`](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) with [permission to push changes to `master` branch](https://help.github.com/en/articles/enabling-branch-restrictions).
```

Comment:
I think we can still improve this message. It suggest that users have to create this token manually in this case, but the default one already has these permissions. Do you think we can rephrase it somehow, so it's more clear?

Reply:
Hum, good point.. I'll think about it..

Reply:
The problem with the provided `GITHUB_TOKEN` is that it cannot push to master if you have branch protection enabled. For that you need to create a personal access token.

You can rephrase it and say 

> To use it you'll need to [generate a personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) with [permission to push changes to `master` branch](https://help.github.com/en/articles/enabling-branch-restrictions). Store the personal access token as a secret, then set the `GITHUB_TOKEN` environment variable from it (instead of from the provided `GITHUB_TOKEN` secret)

```yml
        env:
          GITHUB_TOKEN: ${{ secrets.MY_PERSONAL_ACCESS_TOKEN }}
```

Reply:
Thanks @gr2m , I changed with your changes. Please, take a look what do you think guys. @merlinnot